### PR TITLE
[web] omit inferred types in web_ui

### PR DESCRIPTION
**Pull request by yjbanov**
Add the `omit_local_variable_types` lint and call `dart fix --apply` in `lib/web_ui`.

### CODE REVIEW

**Comment by jtmcdole** [comment]
I think this is LGTM; but I'll leave it to you guys since its `web_ui`.

File: lib/web_ui/analysis_options.yaml [hunk 1]
```diff
@@ -16,3 +16,4 @@ linter:
     public_member_api_docs: false
     unawaited_futures: true
     use_setters_to_change_properties: false
+    omit_local_variable_types: true
```
Comment (kevmoo) [inline lib/web_ui/analysis_options.yaml:19]: might want to be clean and sort these!

**Comment by yjbanov** [comment]
> I think this is LGTM; but I'll leave it to you guys since its `web_ui`.

Still feeling uneasy about this. Had I landed this PR, [this revert](https://github.com/flutter/engine/pull/53679) would have been a nightmare :/

**Comment by kevmoo** [comment]
There is no way through this without dealing with the potential mess

**Pull request closed without merging**